Automattic/jetpack
# Protect: WAF UI header off when no rules settings enabled

#### Description
Update WAF UI header to consider firewall off when no rules are turned on.

## Proposed changes:

* Updates the `currentStatus` variable in `ConnectedFirewallHeader` component to check `jetpackWafAutomaticRules` and `jetpackWafIpList` settings when determining whether to display as `on` or `off`.
* Adds handling so the `automaticRulesAvailable` initial state property is properly updated.

### Other information:


## Jetpack product discussion


- 1201069996155224-as-1203709213079829

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout this branch
* Start up Jurassic Tube
* Ensure you have a Scan upgrade
* Enabled Protect and proceed the Firewall tab
* Verify that when all settings are disabled the header display a message indicating that the Automatic Firewall is off
* Enable Automatic rules and/or Manual rules and ensure that in all combinations of the two the header message shows that the Automatic Firewall is on
* Verify that the WAF continues to function as expected
